src/lfn.c

```diff
@@ -482,6 +483,28 @@ LFNMergePath(LPTSTR lpMask, LPTSTR lpFile)
    return TRUE;
 }
 
+/* WFCopySymlink
+ *
+ *  Copies symbolic links 
+ */
+DWORD
+WFCopySymlink(LPTSTR pszFrom, LPTSTR pszTo, DWORD dwFlags, DWORD dwNotification)
+{

```

Comment:
This function only 'copies szFrom to szTo' if pszFrom is a symbolic link.  I think the name should include the conditionality; e.g., WFCopyIfSymlink.